docs/source/api/index.md

```diff
@@ -2,24 +2,19 @@
 
 ## Top level objects
 ```{eval-rst}
-.. currentmodule:: pymmmc
+.. currentmodule:: pymc-marketing
 .. autosummary::
   :toctree: generated/
 
-
-  ContNonContract
-  ContContract
 ```
 
 ## Modules
 ```{eval-rst}
-.. currentmodule:: pymmmc
+.. currentmodule:: pymc-marketing
```

Comment:
Same comment here. Here I add the suggestion as this section definitely needs to be there

```suggestion
.. currentmodule:: pymc_marketing
```